acceptance/openstack/cts/v1/trackers_test.go

```diff
@@ -12,56 +14,46 @@ func TestTrackersLifecycle(t *testing.T) {
 	client, err := clients.NewCTSV1Client()
 	th.AssertNoErr(t, err)
 
-	bucketName := createOBSBucket(t)
+	bucketName := cts.CreateOBSBucket(t)
 	t.Cleanup(func() {
-		deleteOBSBucket(t, bucketName)
-	})
-
-	smn := createSMNTopic(t)
-	t.Cleanup(func() {
-		deleteSMNTopic(t, smn.TopicUrn)
+		cts.DeleteOBSBucket(t, bucketName)
 	})
 
 	t.Logf("Attempting to create CTSv1 Tracker")
-	createOpts := tracker.CreateOptsWithSMN{
+	ctsTracker, err := tracker.Create(client, tracker.CreateOpts{
 		BucketName: bucketName,
-		SimpleMessageNotification: tracker.SimpleMessageNotification{
-			IsSupportSMN:          false,
-			TopicID:               smn.TopicUrn,
-			IsSendAllKeyOperation: false,
+		Lts: tracker.CreateLts{
+			IsLtsEnabled: pointerto.Bool(true),
 		},
-	}
-	ctsTracker, err := tracker.Create(client, createOpts).Extract()
+	})
+
 	t.Cleanup(func() {
 		t.Logf("Attempting to delete CTSv1 Tracker: %s", ctsTracker.TrackerName)
-		err := tracker.Delete(client).ExtractErr()
+		err := tracker.Delete(client)
```

Comment:
this is really works? it should delete by name: DELETE /v1.0/{project_id}/tracker?tracker_name=system